atlas-commerce-connector/atlas-commerce-blocks.mdx

```diff
@@ -0,0 +1,30 @@
+---
+slug: /atlas-commerce-connector/atlas-commerce-blocks
+title: Atlas Commerce Connector for BigCommerce - Atlas Commerce Blocks
+description: Learn how to build content with BigCommerce products in the WordPress editor.
+---
+
+**Note**: These tutorials assume that you have already [configured the plugin with your BigCommerce API Account credentials](connecting-bigcommerce.mdx) and [performed a product sync](performing-sync.mdx).
+
+Atlas Commerce Blocks lets you place product content synced from BigCommerce in your WordPress content using the WordPress editor data from BigCommerce into WordPress. Using the familiar blocks editorr, you can build pages and posts with product content.
```

Comment:
1. You know, I'm not native speaker, this first sentence sounds really hard to me. Any chance we could break it apart? 

Something like
> Atlas Commerce Blocks is a plugin that <...>. It is possible thanks to <...>. 

2. `editorr` typo

Reply:
Ahh nice one! Cheers Mike